cfme/infrastructure/pxe.py

```diff
@@ -678,6 +678,68 @@ class PXESystemImageTypeEditView(PXESystemImageTypeForm):
     cancel = Button('Cancel')
 
 
+class SystemImage(Updateable, Navigatable):
```

Comment:
**REQUIRED**
Please make it as a collection.

Reply:
@quarckster That's actually one thing I'm not so sure about. I don't think that SystemImage really fits into definition of entity, since you cannot really create or delete one.

Reply:
It doesn't matter, you can instantiate an entity from a collection. Also we should'n use `Navigatable` directly.

Reply:
@quarckster Okay, I'll look into that once I have some time. I suppose it will take some time since I never created one.